internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -143,10 +138,15 @@ Resources:
     Type: AWS::ECS::Service
     DependsOn:
     {{- if .ALBEnabled}}
-     - WaitUntilListenerRuleIsCreated
```

Comment:
Stupid question again: why do we now not need the wait condition?

Reply:
Because `WaitUntilListenerRuleIsCreated` is used for conditional dependency. In this PR I use go template to handle the logic so that we can ditch this.